dev/airflow-pr

```diff
@@ -263,6 +263,23 @@ def merge_pr(pr_num, target_ref, title, body, pr_repo_desc, local):
             'continue.')
         click.pause(click.style(msg, fg='blue', bold=True))
 
+        # The user might have removed "Closes #XXXX" from the commit message
+        # so we add it back to make sure GitHub closes the PR.
+        close_msg = "Closes #{}".format(pr_num)
```

Comment:
If you just have:

    close_msg = "closes #{}".format(pr_num)

Then no lower() is needed below.